build.gradle

```diff
@@ -359,6 +358,13 @@ task switchGoogleServicesCommcare(type: Copy) {
     description = 'Makes build use normal google-services.json file'
     println project.ext.COMMCARE_APP_ID
 
+    /**
+     * This is an awkward workaround for a Windows build error that occurs when you try
+     * to copy something into the root directory (containing the .gradle folder). Don't
+     * change it without testing on a windows machine
+     */
+    outputs.files.setFrom(file("$projectDir/google-services.json"))
```

Comment:
Something has changed in Gradle 3.x that causes this to not work
Jenkins shows the following error:
```
15:08:48 * Where:
15:08:48 Build file '/home/jenkins/workspace/commcare-android-pr-job-builder/commcare-android/build.gradle' line: 366
15:08:48 
15:08:48 * What went wrong:
15:08:48 A problem occurred evaluating root project 'commcare-android'.
15:08:48 > No signature of method: org.gradle.api.internal.tasks.DefaultTaskOutputs$1.setFrom() is applicable for argument types: (java.io.File) values: [/home/jenkins/workspace/commcare-android-pr-job-builder/commcare-android/google-services.json]
15:08:48   Possible solutions: sort()
```

Does it work on Windows?